docs/client-specifications.rst

```diff
@@ -244,26 +244,55 @@ Signature Verification
 
 Clients SHOULD verify the integrity of the downloaded data.
 
+Signature verification allows to guarantee:
+
+- authenticity (+integrity) of data obtained from the server;
+- that client local data was not tempered between two syncs.
+
 .. note::
 
-    Although Gecko on desktop is not exposed to the same risks as on mobile where applications and data are jailed, verifying signatures is a keystone in the chain of trust for data. It is the only way to guarantee the authenticity (and/or integrity) of the data.
+    Although Gecko on desktop is not exposed to the same risks as on mobile where applications and data are jailed, verifying signatures is a keystone in the chain of trust for data that we pull from remote servers.
+
+.. image:: images/client-specifications-signature-verification.svg
```

Comment:
The text in this svg gets all cut off for me in my browser, my rst viewer, and in boxySVG. :thinking: 
Is this a me problem? Example screenshots.
![image](https://github.com/user-attachments/assets/30804878-98b1-4b3b-a9dd-845172b4725a)
![image](https://github.com/user-attachments/assets/b47eb084-426a-4c30-97be-e488d4075588)

Reply:
![Screenshot 2024-11-28 at 11-41-46 Client Specifications — Remote Settings documentation](https://github.com/user-attachments/assets/3a247748-8536-48d7-ab08-44f5d65e00d4)